src/test/java/io/strimzi/kafka/bridge/ConfigRetrieverTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge;
+
+import io.strimzi.kafka.bridge.config.BridgeConfig;
+import io.strimzi.kafka.bridge.config.ConfigRetriever;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.strimzi.kafka.bridge.config.BridgeConfig.BRIDGE_ID;
+import static io.strimzi.kafka.bridge.config.KafkaConfig.KAFKA_CONFIG_PREFIX;
+import static io.strimzi.kafka.bridge.config.KafkaConsumerConfig.KAFKA_CONSUMER_CONFIG_PREFIX;
+import static io.strimzi.kafka.bridge.config.KafkaProducerConfig.KAFKA_PRODUCER_CONFIG_PREFIX;
+import static io.strimzi.kafka.bridge.http.HttpConfig.HTTP_HOST;
+import static io.strimzi.kafka.bridge.http.HttpConfig.HTTP_PORT;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Unit tests for the ConfigRetriever class
+ */
+public class ConfigRetrieverTest {
+
+    @Test
+    public void testApplicationPropertiesFile() throws IOException {
```

Comment:
Is this test actually testing that the ConfigRetriever correctly reads the properties file and gets the right value out? If I changed the ConfigRetriever class to always return "foo" as the value for every key I think this test would still pass right? It's testing whether the BridgeConfig class is using the ConfigRetriever, rather than testing that the ConfigRetriever correctly reads config I think

Reply:
Good point. I think you are right. I will change the assertions to test that the bridge configuration (got through the retriever) actually matches the values in the application.properties file (which means that config retriever was able to read the file correctly as well).

Reply:
@katheris I updated the test. Does it look better now?

Reply:
Yeah that looks better